lib/swoc/include/swoc/Vectray.h

```diff
@@ -232,7 +232,7 @@ template <typename T, size_t N, class A = std::allocator<T>> class Vectray {
 
 template <typename T, size_t N, typename A> Vectray<T, N, A>::Vectray() {}
 
-template <typename T, size_t N, class A> Vectray<T, N, A>::Vectray(Vectray::size_type n, allocator_type const &alloc) : Vectray() {
+template <typename T, size_t N, class A> Vectray<T, N, A>::Vectray(Vectray::size_type n, allocator_type const &) : Vectray() {
```

Comment:
I was told to not change code in the `lib` directory because it contains 3rdparty libraries.
However, this was the only error reported from there and I don't think it could have been silenced via `CMakeLists.txt` configuration because this is a template class which is not built in the library but directly included and used from outside.
I think, it was either changing this piece of code or leaving the warning suppression at the highest level.

Reply:
Would you make a special PR for the libswoc change? I think there's value in keeping a separation between the core changes and the libswoc changes in case people want to work with the libswoc commit history.

Reply:
Done.